### DISCUSSION TITLE
无效：无法复现 uosc菜单字体问题

### DISCUSSION THREAD

**Discussion by SH-Youth** [hooke007/mpv_PlayKit#299] [回收]
![image](https://user-images.githubusercontent.com/89515271/232226010-b11d24d8-dd28-4d83-b357-0e760e900204.png)
如图。不知是否是缺失字体，也不知uosc默认字体是什么。
在https://github.com/hooke007/MPV_lazy/blob/main/portable_config/scripts/uosc/main.lua中更改custom_font也没有作用。
希望得到帮助，谢谢(。﹏。)

**Comment by hooke007** [comment]
信息太少。
没有默认字体，默认用的是—osd-font，win10简中系统下自然是雅黑
而且为什么要直接修改脚本而不是配置文件

**Reply by SH-Youth** [reply]
感谢回复，已解决
未设置osd-font时默认使用的并非雅黑而是懒人包fonts文件夹下的Ubuntu Mono
此时无论直接修改脚本还是配置文件uosc都是跟着一起用Ubuntu Mono
在设置osd-font后修改uosc.conf的custom_font便可生效
不清楚为什么uosc在用Ubuntu Mono时现实汉字有问题

**Reply by hooke007** [reply]
无法复现。你的系统字体设置可能存在问题
osd-font的默认值是sans-serif，即便不显式指定其他字体也不会fall back到fonts里的ubuntu monoo

ubuntu mono里没有中文字符

**Reply by hooke007** [reply]
推送一个新提交同步与微修复 https://github.com/hooke007/MPV_lazy/commit/c3b02e581ec39da80c8a711a2a8e8f98405a3894 。不确定是否能解决你的问题，如果还是渲染异常只能是你那边的环境问题了
